src/main/java/org/rumbledb/parser/JsoniqLexer.java

```diff
@@ -1,18 +1,16 @@
-// Generated from ./src/main/java/sparksoniq/jsoniq/compiler/parser/Jsoniq.g4 by ANTLR 4.7
+// Generated from ./src/main/java/org/rumbledb/parser/Jsoniq.g4 by ANTLR 4.7
 
 // Java header
-package sparksoniq.jsoniq.compiler.parser;
+package org.rumbledb.parser;
 
-import org.antlr.v4.runtime.CharStream;
 import org.antlr.v4.runtime.Lexer;
-import org.antlr.v4.runtime.RuntimeMetaData;
-import org.antlr.v4.runtime.Vocabulary;
-import org.antlr.v4.runtime.VocabularyImpl;
-import org.antlr.v4.runtime.atn.ATN;
-import org.antlr.v4.runtime.atn.ATNDeserializer;
-import org.antlr.v4.runtime.atn.LexerATNSimulator;
-import org.antlr.v4.runtime.atn.PredictionContextCache;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.Token;
+import org.antlr.v4.runtime.TokenStream;
+import org.antlr.v4.runtime.*;
+import org.antlr.v4.runtime.atn.*;
 import org.antlr.v4.runtime.dfa.DFA;
+import org.antlr.v4.runtime.misc.*;
```

Comment:
I have a tiny remark here. I noticed that the wildcard imports came back with these changes. I am wondering if these files are generated with the wildcard format by ANTLR or if your IDE of choice is collecting them together. 

Furthermore Google Java Style Guide, which I believed we were following with spotless (I may be wrong), seems to [not permit wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements).

In any case, since the files are auto generated this is definitely a minor issue but it just caught my attention.

Reply:
This directly came from ANTLR, followed by applying spotless.

Are you aware of an ANTLR configuration parameter that I should set to prevent wildcards?

Thanks!